README.md

```diff
@@ -1,2 +1,41 @@
-# openfisca-italy
-Italian tax and benefit system for OpenFisca.
+# Openfisca-italy
+[![Build Status](https://travis-ci.org/openfisca/openfisca-italy.svg?branch=master)](https://travis-ci.org/openfisca/openfisca-italy)
+## [EN] Introduction
+OpenFisca is a versatile microsimulation free software. This repository contains the OpenFisca model of the French tax and benefit system. Therefore, the working language here is Italian. You can however check the [general OpenFisca documentation](http://openfisca.org/doc/) in English!
+## [IT] Introduzione
+OpenFisca è un software gratuito di microsimulazione versatile. Questa repository contiene il modello del sistema Italiano di tasse e benefici. Perciò, la lingua con cui lavoreremo sarà l'italiano. Si può comunque consultare la [documentazione](http://openfisca.org/doc/) scritta in inglese.
+## Feature
+### Obiettivi da raggiungere 
+* Costruire un sistema di tasse e benefici completo per l'Italia, definendo ogni singola Variabile, Parametro, Riforma,Entità e Formula;
+* Implementare un software grafico dalle diverse funzionalità che possa essere utilizzato dalla pubblica amministrazione per misurare l'impatto che porterebbe al sistema Italiano il cambiamento di una Variabile/Parametro/Riforma tramite un sistema di simulazioni.
+## Prerequisiti per l'installazione
+### Sistemi operativi supportati
+* Distribuzioni Gnu/Linux (in particolare Debian e Ubuntu);
+* Mac OS X ;
+* Windows;
+### Strumenti necessari
+* [Git](https://git-scm.com/)
+* [Python v. 2.7](https://www.python.org/download/releases/2.7/)
+* [Pip](https://pip.pypa.io/en/stable/installing/)
+* [Numpy](https://pypi.python.org/pypi/numpy)
+* [Pew](https://pypi.python.org/pypi/pew/)
+## Installazione
+Per ora non esiste ancora una versione stabile di open-fisca-italy.
+## Iniziare a contribuire
+Per iniziare a contribuire al progetto open-fisca-italy leggi la seguente [linea guida](http://openfisca.org/doc/contribute/guidelines.html) che spiega nel dettaglio le modalità con le quali effettuare le operazioni. Dopodiché esegui i seguenti step:
+* Aprire un terminale; 
+* Spostarsi nella directory nella quale si vuole clonare il progetto;
+* Clonare il progetto nella directory corrente utilizzando il comando:
+```bash
+git clone https://github.com/openfisca/openfisca-italy
+```
+
+Questo è un esempio di output <br />
+![Cmd-all](https://lh3.googleusercontent.com/Bk2NwOH-codW6t6lfR1njHkbl5Ftb345UrRa-JXBk_xtVSdYy9zAylqMdCUxCgyiQXnfByXCm8RX4a0ALWq4P-efQWMJ1ab2nlc=w979-h512-rw-no)
```

Comment:
The link downloads a file and I'm affraid to open it. What did you want your readers to see ?

Reply:
The link you scared to open is an example output of cloning OpenFisca-italy in command prompt. I don't know why git force you to download it, this is the preview that git give me about this image.
![image](https://user-images.githubusercontent.com/36449327/37646142-190ec2e0-2c29-11e8-90cd-d1a98c623c89.png)

Reply:
<img width="356" alt="capture d ecran 2018-03-20 a 11 23 52" src="https://user-images.githubusercontent.com/13916213/37649133-347e964c-2c31-11e8-9f3d-51d412b4a2a0.png">

This is what it looks like in the preview on my end.